draft-ietf-quic-multipath.md

```diff
@@ -528,6 +528,20 @@ proposed and implemented, notably for Multipath TCP. A companion draft
 {{I-D.bonaventure-iccrg-schedulers}} provides several general-purpose
 packet schedulers depending on the application goals.
 
+# Recovery
+
+Simultaneous use of multiple paths enables new techniques to cope with packet losses.
+In {{QUIC-TRANSPORT}}, a lost packet can either be abandoned or retransmitted over the
+same path. However, when multiple paths are available QUIC implementation can use
+different retransmission strategies to cope with losses.
+
```

Comment:
That's not actually true. In {{QUIC-TRANSPORT}}, packets sent on a path before a migration can be resent on the new path after migration. Or, actually, "frames of packets sent on an old path and presumed lost may be resent in new packets on a new path after migration." I think that rather than making comments on QUIc-TRANSPORT, we should just keep the last sentence, "When multiple paths are available QUIC implementation can use
different retransmission strategies to cope with losses."

Reply:
Agreed.